UnrealPlugin/Source/PlayfabGSDK/Public/PlayfabGSDK.h

```diff
@@ -25,7 +25,7 @@ DECLARE_LOG_CATEGORY_EXTERN(LogPlayfabGSDK, Log, All);
 
 
 
-class PLAYFABGSDK_API FPlayfabGSDKModule : public IModuleInterface
```

Comment:
Is there a particular reason to remove this API tag? We do things similarly to this in the UnrealEngineMarketplacePlugin here: https://github.com/PlayFab/UnrealMarketplacePlugin/blob/master/4.26/PlayFabPlugin/PlayFab/Source/PlayFab/Classes/PlayFabAdminAPI.h#L27

Since you already declare GSDK in the name, I think it should be safe to make the change.

Reply:
This class doesnt need to be accessible to other modules. Other modules should use all the functions in the UGSDKUtils class instead